owasp-top10-2017-apps/a3/snake-pro/app/main.go

```diff
@@ -58,6 +58,7 @@ func main() {
 
 	echoInstance := echo.New()
 	echoInstance.HideBanner = true
+	echoInstance.StartTLS(":10033", "./cert.pem", "./key.pem")
```

Comment:
When we are using `echo`, we must first set all routes and the start the server. This line should be near line 89: 

```go
echoInstance.Logger.Fatal(echoInstance.StartTLS(":10033", "./cert.pem", "./key.pem"))
```